src/stores/base.js

```diff
@@ -21,6 +21,7 @@ const useBaseStore = defineStore('base', {
 			port: '/dev/zwave',
 			allowBootloaderOnly: false,
 			commandsTimeout: 30,
+			sendToSleepTimeout: 250,
```

Comment:
Do you need to set a default? This would override any changes I do in the driver, right?

Reply:
Nope I don't need to as long as you accept undefined as value.

Reply:
yep:
![grafik](https://github.com/zwave-js/zwave-js-ui/assets/17641229/30a92b14-4a03-41b1-ab50-3e946e93e95d)